gamebook/scene.py

```diff
@@ -66,7 +66,8 @@ def next_scene_draw(self, option_user_key):
         option = self.get_option_by_user_input(option_user_key)
 
         if not option:
-            raise Exception("this option does not exist in this scene")
+            raise Exception("the user input_recieved",
```

Comment:
Try using a more specific `Exception` (like ValueError) and match the correct parameters.